README.md

```diff
@@ -4,6 +4,29 @@ You will need [Leiningen][1] 2.0 or above installed.
 
 [1]: https://github.com/technomancy/leiningen
 
+### Getting up and running in Debian (wheezy)
```

Comment:
No worries!  I didn't mention installation of `postgresql-jdbc-java` but I figured that would get installed with something along the way.  Given that jessie came out last week, I'll have to update this when I upgrade.  Hopefully more modern versions of the necessary packages are included.